rllib/core/testing/utils.py

```diff
@@ -26,6 +25,9 @@
 Optimizer = Union["tf.keras.optimizers.Optimizer", "torch.optim.Optimizer"]
 
 
+DEFAULT_POLICY_ID = "default_policy"
```

Comment:
Why don't we import this from `policy.py` here?

Reply:
I want to avoid mixing policy code in the new stack.